### ISSUE TITLE
Line breaks in CSV files are ignored

### ISSUE THREAD

**Issue by danrot** [mikefarah/yq#1974]
**Describe the bug**

I have exported a CSV file from a google spreadsheet, whereby some cells contain line breaks. When reading that file using `yq` these line breaks are removed, although I'd like to keep them. This happens no matter which output format I choose using the `-o` command line flag.

Version of yq: 4.42.1
Operating system: mac
Installed via: homebrew

**Input Yaml**
test.csv:
```csv
Header1,Header2
"Some data with a
line break",Some data without a line break
```

**Command**
```csv
yq test.csv
```

**Actual behavior**

```csv
Header1,Header2
Some data with a line break,Some data without a line break
```

**Expected behavior**

```yaml
Header1,Header2
"Some data with a
line break",Some data without a line break
```

**Additional context**

As said before, this also happens when converting the content to YAML, JSON, ... In my use case I would need a JSON output including `\n`s to represent line breaks, although I'd be open to solve that differently. However, by just ignoring the line breaks in this case it is impossible to solve my current task.

**Comment by mikefarah** [comment]
Thanks for raising, will fix in the next release

**Comment by mikefarah** [comment]
Fixed in v4.43.1 :)
